include/sdf/PrintConfig.hh

```diff
@@ -90,6 +104,20 @@ namespace sdf
     /// \return True if 'this' == _config.
     public: bool operator==(const PrintConfig &_config) const;
 
+
+    /// \brief Implementation function for SetRotationSnapToDegrees. It sets
+    /// the option for printing pose rotation in degrees as well as
+    /// snapping the rotation to the desired interval, with the provided
+    /// tolerance.
+    /// \param[in] _interval Degrees interval to snap to, this value must be
+    /// larger than 0, and less than or equal to 360.
+    /// \param[in] _tolerance Tolerance which snapping occurs, this value must
+    /// be larger than 0, less than 360, and less than the provided interval.
+    /// \param[out] _errors Vector of Errors.
+    /// \return True, unless any of the provided values are not valid.
+    private: bool SetRotationSnapToDegreesImpl(unsigned int _interval,
```

Comment:
Would it be possible to not add this member function and instead use the overload with the `Errors` object?

Something along the lines of:
```
bool PrintConfig::SetRotationSnapToDegrees(unsigned int _interval) {
  sdf::Errors errors;
  bool result = this->SetRotationSnapToDegrees(_interval, errors);
  sdferr << errors;
  return result;
}
```

Reply:
Yes, done in https://github.com/gazebosim/sdformat/pull/1098/commits/a498d9e371ab03a6a4a9b59ef66869eb7369b06d.